code/modules/mob/mob.dm

```diff
@@ -984,7 +984,10 @@ GLOBAL_LIST_INIT(slot_equipment_priority, list( \
 	return gender
 
 /mob/proc/is_muzzled()
-	return 0
+	return FALSE
+
+/mob/proc/is_facehugged()
+	return FALSE
```

Comment:
I don think this needs to be scoped to mob.

Reply:
Will change to /mob/living. Facehuggers can specifically attach to corgis, so its scoping has to at least cover simplemobs.

Reply:
Scoped to living. Also changed the scoping on the Swede disability's mob param to mob/living, since a check is made for muzzles/facehuggers (to make sure that you can't say "bork bork bork!" while muzzled), and dead people can't get disabilities.